pkg/plugin/standalone/v1/logger.go

```diff
@@ -33,6 +33,14 @@ var hclogZerologLevelMapping = map[hclog.Level]zerolog.Level{
 	hclog.Warn:    zerolog.WarnLevel,
 	hclog.Error:   zerolog.ErrorLevel,
 }
+var zerologHclogLevelMapping = func() map[zerolog.Level]hclog.Level {
+	// reverse hclog->zerolog level mapping
+	m := make(map[zerolog.Level]hclog.Level, len(hclogZerologLevelMapping))
```

Comment:
so IIUC you created a function instead of just creating a reversed map like manually, because you want to have one reference? instead of editing two maps in the future u would edit one, right? or is there another reason?